# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#Define the result
uint32 trajectory_id
float32[] resulting_trajectory
